src/common/components/traffic-lights/traffic-light.css

```diff
@@ -19,23 +18,24 @@ $signal-colour-default: $light-grey;
   line-height:1.286;
   position: relative;
   z-index:10;
+  border: 2px solid $signal-colour-default;
 }
 
 .signal::before {
   content: "\0000a0"
 }
 
 .signal::after {
-  content:'';
+  content: '';
   display: block;
   height: 4px;
-  width: 850%;
+  width: 320px;
   background: $signal-colour-default;
   position: absolute;
-  top:50%;
-  left:106%;
-  z-index:-1;
-  margin-top:-2px;
+  top: 50%;
+  left: 30px;
```

Comment:
Yeah, this was a hurried attempt at making things vaguely responsive that failed.